addons/kotsadm/alpha/tmpl-secret-cluster-token.yaml

```
@@ -1,8 +1,8 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: kotsadm-auto-create-cluster-token
+  name: kotsadm-cluster-token
   labels:
     kots.io/kotsadm: 'true'
 stringData:
-  token: ${AUTO_CREATE_CLUSTER_TOKEN}
+  token: ${CLUSTER_TOKEN}
```

the key and the secret name are both `kotsadm-cluster-token` in kots
```suggestion
  kotsadm-cluster-token: ${CLUSTER_TOKEN}
```